dist/openshift/graph-data-sync.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: cincinnati
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app: cincinnati-graph-data-sync
+      name: cincinnati-graph-data-sync
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: cincinnati-graph-data-sync
+      strategy:
+        type: Recreate
+      template:
+        metadata:
+          labels:
+            app: cincinnati-graph-data-sync
+          name: cincinnati-graph-data-sync
+        spec:
+          containers:
+            - args:
+                - -c
+                - |
+                  set -euo pipefail
+
+                  pip install --user PyYAML
+
+                  cd /tmp
+                  while sleep 600
```

Comment:
Perhaps `git-sync` sidecar would be a better fir here? We could be notified that git repo has changed via inotify / custom webhook.

This version is fine as a PoC though

Reply:
I'm fine with a sync sidecar if you want.  But the goal here is just to give us something that works for a week or three until the Rust graph builder learns the format, so I'm probably not spending more time on this myself ;)

Reply:
+1, I'm fine with reworking this later